PMurphy1940/HandPickt
# Archives

# Description
The Archive view is fully functional. User archived plants are displayed in a vertically scrolling view.

## Type of change


# Testing Instructions

 The user may ```Inspect``` or ```Remove``` a plant from the main list. Inspect will bring the user to the plants details view which will show an expanded card with full details on the plant. The user may edit the comments. Clicking anywhere on the field will active edit mode. Editable fields are indicated by the 'Pencil and Note' icons. Clicking the check after editing will turn off the edit field, but changes are retained but not yet saved. If any fields are edited, a ```Save Changes``` button will display at the bottom of the view, just above the Nav Bar. The ```Back``` button will close the expanded view and discard any changes. ```Remove``` will do same as above.

# Checklist:
